templates/modules/b2b-account/quotes/quote-history.hypr.live

```diff
@@ -90,6 +90,23 @@
 
                               {% if change.path %}
                               <tr class="space-before">
+                                <td>
+                                  {% if change.type =='Add' %} {{labels.added}} {% elif change.type =='Remove' %} {{labels.removed}} {% else %} {{labels.updated}} {% endif %}
+                                  {{ changePath[0] }}
+                                </td>
+                              </tr>
+                              <tr>
+                                {% set shippingMethodName=change.fields|findwhere("name", "ShippingMethodName") %}
+                                <td>Shipping Methods</td>
+                                <td class="address">
+                                  {% include "modules/b2b-account/quotes/quote-history-field" with name=shippingMethodName value=shippingMethodName.oldValue %}
+                                </td>
+                                <td class="space"></td>
+                                <td class="address">
+                                  {% include "modules/b2b-account/quotes/quote-history-field" with name=shippingMethodName value=shippingMethodName.newValue %}
```

Comment:
this looks wrong to me, you are passing `name=shippingMethodName` which should be `name=shippingMethodName.name`. see how we have done this in other places.

Reply:
Done